src/main/resources/languages/english.xml

```diff
@@ -736,6 +736,11 @@
 		<value>Error in communicating with the robot.</value>
 		<hint>text</hint>
 	</string>
+	<string>
+		<key>PlotterControls.CommunicationFailure</key>
+		<value>No answer from the robot, please check the USB cable, speed settings...</value>
```

Comment:
what does "..." mean?  as a novice user this would be confusing.

Reply:
what do you propose ? I can remove the three dots for clarity if you want.

Reply:
I don't immediately have a good suggestion.  Let's talk about it, maybe see what users would like.